schools/aggie.yml

```diff
@@ -2,6 +2,8 @@ version: 0.1
 name: Aggie
 label: TAMU - Texas A&M University
 studentEmailDomain: tamu.edu
+entitlements:
+  course-search: false
```

Comment:
Do we currently restrict course search to the Corps?

Reply:
Noe

Reply:
So are we intending to not allow course search on aggie?

Reply:
That would be a typo 😄